sites/footers/index.json-ld

```diff
@@ -131,6 +131,18 @@
 				"language": "fr",
 				"path": "contextual-example-fr.html"
 			}
+		],
+		"reports": [
+			{
+				"title": "Accessibility assessment #1 - Footer",
+				"language": "en",
+				"path": "reports/ally-1-en.hbs"
```

Comment:
does this needs to be reports/ally-1-en.html

Reply:
Yes it does.

@BrahimMahadi this URL need to be
```suggestion
				"path": "reports/a11y-1-en.html"
```

Reply:
Fixed in commit "Update index.json-ld"